event/src/unvalidated/event.rs

```diff
@@ -50,6 +50,7 @@ impl<D> From<signed::Envelope> for RawEvent<D> {
 
 /// Time event
 #[derive(Debug, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)] // TODO Should this be here?
```

Comment:
Time events feel different than data and init events because they'll generally be created by the protocol (C1 nodes or the CAS) rather than by users of the protocol.  So maybe it makes sense to be looser in what we expect here so it's easier for us to make additive changes to the events in the future without breaking older nodes?

On the other hand, nothing stops users from making their own TimeEvents and publishing them, in which case issues with field name collisions could still happen.

I'm conflicted, probably leaning slightly towards not adding this directive and keeping validation looser here, but could be easily swayed if anyone thinks it's worth locking it down now.